modules/ROOT/pages/file-name-pattern.adoc

```diff
@@ -1,8 +1,8 @@
 = File Name Patterns for Send Endpoints
 
-Partner Manager enables you to configure the file name patterns to use on the target files delivered to SFTP and AS2 Send endpoints, through the use of property placeholders.
+Using property placeholders, Partner Manager enables you to configure the file name patterns for target files delivered to SFTP and AS2 send endpoints.
```

Comment:
What is a "file name pattern"?
What is a "target file," as opposed to a "file delivered to...endpoint" (the endpoint being the target?)?
Does this mean that by using placeholders, you can configure a file name template that specifies properties and their order in that name?

Reply:
Sorry...I guess all these questions really come under the CU banner and I should be sticking with style....

Reply:
Updated